k8s/pkg/apis/longhorn/v1beta2/engine.go

```diff
@@ -77,6 +77,8 @@ type RebuildStatus struct {
 	State string `json:"state"`
 	// +optional
 	FromReplicaAddress string `json:"fromReplicaAddress"`
+	// +optional
+	AppliedRebuildingMbps int64 `json:"appliedRebuildingMbps"`
```

Comment:
Is AppliedRebuildingMbytesPerSecond better?

Reply:
I also think `AppliedRebuildingMbytesPerSecond` is a clearer and more descriptive field name. I originally shortened it to `AppliedRebuildingMbps` because it felt too long compared to the other fields in `rebuildStatus`

Reply:
Agree with you. Then, to prevent confusion, how about rename it to `AppliedRebuildingMBps`. `Mbps` is for megabits per second.